languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -23467,6 +23467,15 @@ USA
     <!-- Grammar -->
     <!-- ====================================================================== -->
     <category id="GRAMMAR" name="Grammar" type="grammar">
+        <rule id="GOOD-EDUCATED" name="She is good-educated (well-educated)." default="temp_off">
```

Comment:
Wait, one (1) naming convention I forgot to mention:
We tend to use underscores here, not hyphens.
Once that's changed, you may merge this yourself! 👍

Reply:
Even in instances when the hyphen is orthographically called for?

Reply:
Yes